src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -735,7 +735,764 @@ await using (var context = InMemoryContentDbContext(contextId))
         }
 
         [Fact]
-        public async Task GetPublicationTree_LatestData_SingleThemeTopic()
+        public async Task GetPublicationTree_AnyData_SingleThemeTopic()
+        {
+            var publicationA = new Publication
+            {
+                Title = "Publication A",
+                Slug = "publication-a",
+                Summary = "Publication A summary",
+                LegacyPublicationUrl = new Uri("https://legacy.url/")
+            };
+            var publicationB = new Publication
+            {
+                Title = "Publication B",
+                Slug = "publication-b",
+                Summary = "Publication B summary"
+            };
+
+            var theme = new Theme
+            {
+                Title = "Theme A",
+                Summary = "Theme A summary",
+                Topics = new List<Topic>
+                {
+                    new()
+                    {
+                        Title = "Topic A",
+                        Publications = ListOf(publicationB, publicationA)
+                    },
+                },
+            };
+
+            var releaseFiles = new List<ReleaseFile>
+            {
+                new()
+                {
+                    Release = new()
```

Comment:
There are missing type specification warnings on `Release` and `File` here

Reply:
Fixed, I've done it for all the other instances in the file too.